rllib/algorithms/ppo/tf/ppo_tf_learner.py

```diff
@@ -67,7 +68,7 @@ def compute_loss_per_module(
         else:
             mean_kl_loss = tf.constant(0.0, dtype=logp_ratio.dtype)
 
-        curr_entropy = fwd_out["entropy"]
+        curr_entropy = curr_action_dist.entropy()
```

Comment:
For consistency, we should compute all distribution related things inside the loss.
The forward_train should only return the distribution parameters, not create a distribution and thus also not compute distribution-dependent values.